### PR TITLE
Initial commit for datacrafts-io-skeleton gem with cli.

### PR DESCRIPTION
**Pull request by overchind**
Demo: https://youtu.be/ERR7JMWPBqU

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,37 @@
...
+
+### New Rails API application without frontend part.
+
+    $ datacrafts-io-skeleton create APP_NAME
+    $ datacrafts-io-skeleton build APP_NAME
+    $ datacrafts-io-skeleton new APP_NAME
+
+## Development
+
+After checking out the repo, run `bin/setup` to install dependencies. Then, run `rake spec` to run the tests. You can also run `bin/console` for an interactive prompt that will allow you to experiment.
```
Comment (cxspxr) [inline README.md:21]: Is it possible to run setup showing output and navigate to directory automatically with an appropriate notice?

Comment (cxspxr) [reply README.md:21]: Not relevant, got it now

File: templates/rubocop.yml.erb [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+  Enabled: false
+
+Style/Lambda:
+  EnforcedStyle: literal
+
+Style/Documentation:
+  Enabled: false
+
+Style/FrozenStringLiteralComment:
+  Enabled: false
```
Comment (akurashev) [inline templates/rubocop.yml.erb:27]: I think it's better remove this. Highly likely Ruby 3+ will have string literals frozen by default. So it's better to be prepared :)

File: templates/rubocop.yml.erb [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+  Enabled: true
+
+Lint/DuplicateElsifCondition:
+  Enabled: true
+
+Layout/SpaceAroundMethodCallOperator:
+  Enabled: true
+
+Layout/SpaceInsideStringInterpolation:
+  EnforcedStyle: space
```
Comment (akurashev) [inline templates/rubocop.yml.erb:121]: Probably it's better stick to default setting `no_space`

File: templates/rubocop.yml.erb [hunk 4]
```diff
@@ -0,0 +1,256 @@
...
+  EnforcedStyle: no_space
+
+Layout/ClassStructure:
+  Enabled: true
+
+Layout/LineLength:
+  Max: 120
+
+Metrics/AbcSize:
+  Max: 20
```
Comment (akurashev) [inline templates/rubocop.yml.erb:144]: Why 20? Default 15 usually easy enough to support

File: lib/datacrafts_io_skeleton/composer.rb [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+require_relative "./version"
+require_relative "./rails_creator"
+
+module DatacraftsIoSkeleton
+  # Defines all CLI commands, their descriptions, params and aliases.
+  class Composer < Thor
+    desc "create APP_NAME", "This will create new rails app."
+    long_desc <<-CREATE_NEW_APP
+     This command will create new rails app with configured rails_helper.rb and .rubocop.yml files. 
+     All major gems are also included.
```
Comment (DanilParshukov) [inline lib/datacrafts_io_skeleton/composer.rb:11]: Description will be more clear for nondatacrafts developers if we put this text in the Readme file right after the message `Create new apps quickly!`.

File: templates/rubocop.yml.erb [hunk 6]
```diff
@@ -0,0 +1,256 @@
...
+Style/Documentation:
+  Enabled: false
+
+Style/FrozenStringLiteralComment:
+  Enabled: false
+
+Style/ReturnNil:
+  Enabled: true
+
+
```
Comment (DanilParshukov) [inline templates/rubocop.yml.erb:32]: two empty lines

File: templates/spec.rb [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+
+# Checks for pending migrations and applies them before tests are run.
+# If you are not using ActiveRecord, you can remove these lines.
+begin
+  ActiveRecord::Migration.maintain_test_schema!
+rescue ActiveRecord::PendingMigrationError => e
+  puts e.to_s.strip
+  exit 1
+end
+RSpec.configure do |config|
```
Comment (DanilParshukov) [inline templates/spec.rb:40]: It's not critical, but there should be empty line between `end`  and `RSpec.configure do |config|`

**Pull request merged**